packages/core/components/FileDetails/index.tsx

```diff
@@ -157,7 +165,26 @@ export default function FileDetails(props: FileDetails) {
                                         uri={`http://aics.corp.alleninstitute.org/labkey/fmsfiles/image${thumbnailUriPath}`}
                                     />
                                 </div>
-                            )}
+                            ) : fileDetails ? (
```

Comment:
Can you find a way to do this without having a nested ternary?

Reply:
its pretty similar- but would it be better to use && and || like this? 
`
{ displayThumbnail && (
   //display thumbnail
) || (
   fileDetails && (
      //show icon
   )
)}
`

Reply:
Yeah, that feels a little too similar to me (😄)--for me, the readability concerns of a nested ternary come down to over-terness. Here's how I might approach this: above the return statement, use more verbose logic to assign a React element to a variable, then in the JSX expression returned from this function, make use of that variable. From 10,000 ft, that looks like:
```
function MyReactComponent() {

    let thumbnail = null;
    if (something) {
        thumbnail = (
            <div>...</div>
        );
    } else if (somethingElse) {
        thumbnail = (
             <Foo />
        );
    }

     return (
        <>
            ...
            {thumbnail}
        </>
     );
}
```